### ISSUE TITLE
why lr in fc8_kevin is not as large as fc8_pascal ?

### ISSUE THREAD

**Issue by ToTheBeginning** [kevinlin311tw/caffe-cvprw15#19]
They are both new layers, why the lrs are different?

**Comment by kevinlin311tw** [comment]
It mainly depends on the dataset which you would like to fine-tune. In this repo, we empirically set the learning rate and found it works.
